sys/include/net/gnrc/pkt.h

```diff
@@ -106,10 +106,11 @@ typedef struct gnrc_pktsnip {
      *
      * @internal
      */
-    unsigned int users;
+    /* the first three fields *MUST* be identical to iolist_t! */
```

Comment:
I think you can group fields with `@{` `@}` as well.

Reply:
Hm, I thought this to be an implementation detail, thus the comment is not doxygen (```/*```).